lib/phlex/rails/form_builder.rb

```diff
@@ -3,6 +3,17 @@
 class Phlex::Rails::FormBuilder < Phlex::Rails::Decorator
 	define_method :__caller_locations__, ::Object.instance_method(:caller_locations)
 
+	def method_missing(...)
+		output = @object.public_send(...)
+
+		case output
+		when ::ActiveSupport::SafeBuffer
+			@component.raw(output)
+		else
+			output
+		end
+	end
```

Comment:
This is actually the only method we need. What do you say to getting rid of all the others @willcosgrove?